FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -35,9 +36,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    <note>Dimensions indicated below 
+                                        may concern outer dimensions of bookcover or 
+                                        dimensions of folia (not clearly specified in cataloque).</note>
+                                    <dimensions type="outer" unit="mm">                                        
+                                        <height cert="medium">295-305</height>
+                                        <width cert="medium">250-265</width>    
+                                                 </dimensions>
+                                </extent>
+                                <foliation>Foliated in pencil by a recent European hand at the top on the right.</foliation>
+                                <collation>
+                                    <list>
+                                    <item xml:id="q1" n="1">
+                                        <dim unit="leaf">6</dim>
+                                        <locus from="1r" to="6v"/>
+                                        s.l. 1, stub after 6,
+                                        s.l. 2, no stub.
+                                    </item>
+                                    <item xml:id="q2" n="2"> 
+                                        <dim unit="leaf">6</dim>
+                                        <locus from="7r" to="12v"/>
+                                    </item>
+                                    <item xml:id="q3" n="3">
+                                        <dim unit="leaf">9</dim>
+                                        <locus from="13r" to="21v"/>
+                                            s.l. 1, no stub
+                                        <note>Contrary to Marrassini, who considered <locus target="#13"/> separate from 
+                                            <ref target="#q3"/>. The text in this quire seemingly shows no breaks.</note>                      
+                                    </item>
+                                    <item xml:id="q4" n="4">
+                                        <dim unit="leaf">10</dim>
+                                        <locus from="22r" to="31v"/>
+                                    </item>
+                                    <item xml:id="q5" n="5">
+                                        <dim unit="leaf">8</dim>
+                                        <locus from="32r" to="39v"/>
+                                    </item> 
+                                        <item xml:id="q6" n="6">
+                                            <dim unit="leaf">3</dim>
+                                            <locus from="40r" to="42v"/>
+                                            s.l. 1, no stub,
+                                            s.l. 2, stub after 3
+                                            s.l. 3, stub after 2.
```

Comment:
please remove dot after 2

Reply:
done

Reply:
Nein, der Punkt is da, und die Frage unten ist noch nicht beantwortet.